packages/restitution/src/Restitution.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import {
+  withClassDefault,
+  withClassModifier,
+  WithClassModifierOptions,
+} from '@axa-fr/react-toolkit-core';
+import { compose } from 'recompose';
+
+const DEFAULT_CLASSNAME = 'af-restitution__listdef';
+
+type RestitutionBaseProps = {
+  label: string;
+  value?: string;
+  values?: string[];
+  className?: string;
+}
+
+type RestitutionValuesProps = {
+  values: string[];
+}
+
+const RestitutionValues = ({ values }: RestitutionValuesProps) => {
+  const li = values.map((v: string) => (
+    <li key={v} className="af-restitution__listul-item">
```

Comment:
the className shouldn't contain HTML Tag

Reply:
It was already done by someone else, I won't dobreking change in that PR Samuel ^^
I can set up issues ^^

Reply:
ok for me